components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/SAMLSSOConfigServiceImpl.java

```diff
@@ -396,7 +396,7 @@ public SAMLSSOServiceProviderDTO getServiceProvider(String issuer) throws Identi
     private KeyStoreData[] getKeyStores(int tenantId) throws IdentityException {
 
         try {
-            KeyStoreAdmin admin = new KeyStoreAdmin(tenantId, getGovernanceRegistry());
```

Comment:
Check usages of `getGovernanceRegistry()` method

Reply:
Addressed with: https://github.com/wso2-extensions/identity-inbound-auth-saml/compare/92bee1ebbbe81a043ef93cdb323583511ff410f1..9056bb0bd0dac22c3502369a35d94c806c6b3370